src/subclassers.ts

```diff
@@ -34,169 +38,56 @@ type FunctionTaking<TIn> = (param: TIn) => any;
 
```

Comment:
If all this stuff above isn't needed anymore, please remove it - note that its function signature is using the OLD OOP version (no context - assuming this)
- [ ] more residue

Reply:
- [ ] lol and there's a fixme way above that can be removed ;)

Reply:
the changes is in this PR because i can't update your pr i branch of this branch and create a new PR and set destination to this branch so it will be easier to review
https://github.com/captaincaius/hipthrusts/pull/30